_ml-commons-plugin/remote-models/blueprints.md

```diff
@@ -74,7 +74,7 @@ The `actions` parameter supports the following options.
 
 | Field | Data type | Description |
 |:---|:---|:---|
-| `action_type` | String | Required. Sets the ML Commons API operation to use upon connection. As of OpenSearch 2.9, only `predict` is supported. |
+| `action_type` | String | Required. Sets the ML Commons API operation to use upon connection. Valid values are `predict` (to invoke an externally hosted model) and `execute` (to invoke any REST API function). |
```

Comment:
The `execute` action can't be used for model. Model can only use `predict` action